docs/docs-content/clusters/data-center/nutanix/add-nutanix-cloud-account.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: "Add Nutanix Cloud Account"
+title: "Add Nutanix Cloud Account"
+description: "Learn how to add a Nutanix cloud account in Palette."
+hide_table_of_contents: false
+tags: ["data center", "nutanix"]
+---
+
+ Once the Private Cloud Gateway (PCG) is installed, the Nutanix cloud account must be added to Palette. The required fields to add the cloud are pre-defined by the System Administrator during the cloud registration process. These cloud-specific fields vary based on the cloud type.
+
+## Prerequisites
+
+- A Nutanix cloud that a System Administrator has registered with Palette.
+
+
+## Add Cloud Account
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the **left Main Menu**, select **Tenant Settings**. 
+
+3. Next, select **Cloud Accounts** in the **Tenant Settings** menu.
+
+4. Locate Nutanix and click **Add Nutanix Cloud**.  
+
+5. Fill out the following input values and click on **Confirm** to continue. 
+
+  | **Field** | **Description** |
+  |-----------|-----------------|
+  | **Name**| A custom name for the account. |
+  | **Private Cloud Gateway**| This name is auto-filled based on the name you provided when you installed the PCG.|
+  | **NUTANIX_USER**| Prism Central user name.|
+  | **NUTANIX_PASSWORD** | Prism Central password.|
+  | **NUTANIX_ENDPOINT** | Prism Central IP address or fully qualified domain name (FQDN).|
+  | **NUTANIX_PORT** | Specify the default in the ``cloudClusterTemplate.yaml``: 9440 |
+  | **NUTANIX_INSECURE** | Specify the default in the ``cloudClusterTemplate.yaml``: false |
```

Comment:
```suggestion
  | **NUTANIX_USER**| The Prism Central user name.|
  | **NUTANIX_PASSWORD** | The Prism Central user password.|
  | **NUTANIX_ENDPOINT** | The Prism Central IP address or the fully qualified domain name (FQDN) assigned to Prism.|
  | **NUTANIX_PORT** | Specify the default port you assigned in the file **cloudClusterTemplate.yaml**.  The default value is `9440` |
  | **NUTANIX_INSECURE** | Specify the SSL behavior you used in the file **cloudClusterTemplate.yaml**. The default value `false`. |
```
It's unclear from the description that `9440` and `false` are the default values so I made it more explicit.

Reply:
adde suggested text.